controllers/watches_util.go

```diff
@@ -113,6 +113,7 @@ func (wt *watchesForTemplate) addWatchForObject(ctx context.Context, objectRef t
 		Namespace:     objectRef.Namespace,
 	})
 	if err != nil {
+		logger.Info("Failed to start watch for object", "templateKey", wt.templateKey, "objectRef", objectRef, "error", err.Error())
```

Comment:
Why isn't this logged as an error?

Reply:
I had this being logged as error initially and it caused quite some noise due to stack traces. This error is expected to happen in daily business because users will forget to add proper permissions via RBAC, so I would prefer to not clog the logs with stack traces.

What however should be added in the long-term is error reporting into the status of ObjectTemplate and TextTemplate in regard to these errors.